IaC/AKS/README.md

```diff
@@ -486,15 +486,19 @@ Deploy Web Validate to drive consistent traffic to the AKS cluster for monitorin
 
 ```bash
 
-cd $REPO_ROOT/IaC/AKS/cluster/charts
+cd $REPO_ROOT/IaC/AKS/cluster/charts/smoker
```

Comment:
should we use webv instead of smoker?

Reply:
Or, maybe stay with smoker since we're changing the name of the custom webv (presently Loderunner).  I think using smoker, or some other label, as a non-specific name might better than the actual name.  Thoughts?

Reply:
I agree with the use of smoker, makes sense now that we're switching to Loderunner.